openapi/platform/platform.yaml

```diff
@@ -2933,12 +2933,24 @@ paths:
                 comments:
                   type: string
                   description: more description for cancellation of products
+                deactivationType:
+                  type: string
+                  description: |-
+                    `deactivationTypeCancel` Cancel indicates we should turn the item off at the items anniversary date, or commitment date, whichever is later
+
+                    `deactivationTypeImmediate` Immediate indicates we should turn the item off immediately
```

Comment:
```suggestion
                    `immediate` Immediate indicates we should turn the product or service off immediately and it will no longer be available to use.
```